codegen/options.go

```diff
@@ -0,0 +1,54 @@
+package codegen
+
+import (
+	"flag"
+	"github.com/go-errors/errors"
+	"github.com/viant/toolbox"
+	"github.com/viant/toolbox/url"
+	"strings"
+)
+
+type Options struct {
+	Source      string
+	Dest        string
+	Types       []string
+	PoolObjects bool
+	TagName     string
+}
+
+func (o *Options) Validate() error {
+	if o.Source == "" {
+		return errors.New("Source was empty")
+	}
+	if o.Dest == "" {
+		return errors.New("Dest was empty")
+	}
```

Comment:
I'm removing this, it prevents from printing to stdout if no "o" flag is provided.